pkgs/applications/editors/vim/vimacs.nix

```diff
@@ -17,7 +17,7 @@ stdenv.mkDerivation rec {
       --replace '-gvim}' '-@bin@/bin/vim -g}' \
       --replace '--cmd "let g:VM_Enabled = 1"' \
                 '--cmd "let g:VM_Enabled = 1" --cmd "set rtp^=@rtp@" ${vimacsExtraArgs}' \
-      --replace @rtp@ ${vimPlugins.vimacs.rtp} \
+      --replace @rtp@ ${vimPlugins.vimacs} \
```

Comment:
@millerjason No I didn't verify, as I don't use it.

`vimPlugins.vimacs.rtp` and `vimPlugins.vimacs.outPath` were the same before https://github.com/NixOS/nixpkgs/commit/0dbca90d4129b6c4296b66899fd2750584b24c4f, and I was trying to go back to that with #192733. Since `rtp` and `outPath` are the same and this is the only place where `rtp` attribute was used, vimacs shouldn't be affected by this change.